src/components/FilterSortContainer/index.tsx

```diff
@@ -1,11 +1,24 @@
 import classNames from 'classnames'
-import { ComponentProps, FC } from 'react'
+import { ComponentProps, FC, ReactNode } from 'react'
 import styles from './index.module.scss'
 
 export const FilterSortContainerOnMobile: FC<ComponentProps<'div'>> = ({ children, ...elProps }) => {
+  const { left, right } = Array.from(children as Iterable<ReactNode>).reduce<{ left: ReactNode[]; right: ReactNode[] }>(
+    (acc, child, index) => {
+      if (index % 2 === 0) {
+        acc.left.push(child)
+      } else {
+        acc.right.push(child)
+      }
+      return acc
+    },
+    { left: [], right: [] },
+  )
```

Comment:
CSS selector `odd | even` can be used if this is just for layout

Ref: https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child#keyword_values